### PR TITLE
Feature/32200

### PR DESCRIPTION
**Pull request by szafran89**
https://dev.snowdog.pro/issues/32200

### CODE REVIEW

File: Magento_Checkout/layout/checkout_cart_index.xml [hunk 1]
```diff
@@ -5,36 +5,33 @@
...
             >
-                <block class="Magento\Framework\View\Element\RendererList"
-                       name="checkout.cart.item.renderers"
-                       as="renderer.list"
-                />
-                <block class="Magento\Framework\View\Element\Text\ListText"
-                       name="checkout.cart.order.actions"
+                <container name="cart.main.gift.coupon"
+                           as="cart_main_gift_coupon"
+                           label="Shopping Cart Main Container For Fift Card And Coupon"
```
Comment (niemaco) [inline Magento_Checkout/layout/checkout_cart_index.xml]: Typo in label. I think it should be Gift not Fift.

File: Magento_GiftMessage/web/template/gift-message-form.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
+<!-- ko if: isActive() -->
+<div class="gift-message">
+    <div class="gift-options__sub-title">
+        <span data-bind="i18n: 'Gift Message (optional)'"></span>
+    </div>
+    <div class="gift-options-content">
```
Comment (niemaco) [inline Magento_GiftMessage/web/template/gift-message-form.html]: Should be  gift-options__content?

File: Magento_Wishlist/layout/checkout_cart_item_renderers.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd"
+>
+    <body>
+        <referenceBlock name="checkout.cart.item.renderers.default.actions.move_to_wishlist" remove="true"/>
+        <referenceBlock name="checkout.cart.item.renderers.simple.actions.move_to_wishlis" remove="true"/>
```
Comment (niemaco) [inline Magento_Wishlist/layout/checkout_cart_item_renderers.xml]: Probably missing t on the end of attribute name.

File: Magento_Checkout/layout/checkout_cart_index.xml (hunk 1 above)
Comment (szafran89) [reply Magento_Checkout/layout/checkout_cart_index.xml]: fixed

File: Magento_GiftMessage/web/template/gift-message-form.html (hunk 2 above)
Comment (szafran89) [reply Magento_GiftMessage/web/template/gift-message-form.html]: fixed

File: Magento_Wishlist/layout/checkout_cart_item_renderers.xml (hunk 3 above)
Comment (szafran89) [reply Magento_Wishlist/layout/checkout_cart_item_renderers.xml]: fixed

**Pull request merged**